### ISSUE TITLE
Problems when running

### ISSUE THREAD

**Issue by ruimaranhao** [aboul3la/Sublist3r#20]
Any idea why I am getting the following problem?

✘ maranhao@crystal  ~/test/Sublist3r   master  python sublist3r.py -d www.example.com

                 ____        _     _ _     _   _____
                / ___| _   _| |__ | (_)___| |_|___ / _ __
                \___ \| | | | '_ \| | / __| __| |_ \| '__|
                 ___) | |_| | |_) | | \__ \ |_ ___) | |
                |____/ \__,_|_.__/|_|_|___/\__|____/|_|

                 # Coded By Ahmed Aboul-Ela - @aboul3la

[-] Enumerating subdomains now for www.example.com
[-] Searching now in Baidu..
[-] Searching now in Yahoo..
[-] Searching now in Google..
[-] Searching now in Bing..
[-] Searching now in Ask..
[-] Searching now in Netcraft..
[-] Searching now in DNSdumpster..
[-] Searching now in Virustotal..
[-] Searching now in SSL Certificates..
[-] Searching now in PassiveDNS..
Process GoogleEnum-4:
Traceback (most recent call last):
  File "/Users/maranhao/anaconda/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "sublist3r.py", line 202, in run
    domain_list = self.enumerate()
  File "sublist3r.py", line 170, in enumerate
    resp = self.send_req(query, page_no)
  File "sublist3r.py", line 110, in send_req
    return self.get_response(resp)
UnboundLocalError: local variable 'resp' referenced before assignment
Process BaiduEnum-2:
Traceback (most recent call last):
  File "/Users/maranhao/anaconda/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "sublist3r.py", line 202, in run
    domain_list = self.enumerate()
  File "sublist3r.py", line 170, in enumerate
    resp = self.send_req(query, page_no)
  File "sublist3r.py", line 110, in send_req
    return self.get_response(resp)
UnboundLocalError: local variable 'resp' referenced before assignment

**Comment by ruimaranhao** [comment]
ps - I am running on a macOS

**Comment by aboul3la** [comment]
This should be fixed now in commit a6c55a0a0cee69869c12048c45d44aba89dbe422. 
Kindly update your version now using "git pull" command or clone the repo again.
Thanks

**Comment by pr0letariat** [comment]
I just cloned the repo again but unfortunately this issue still exists.

```
HTTPSConnectionPool(host='dnsdumpster.com', port=443): Read timed out. (read timeout=25)
Process DNSdumpster-8:
Traceback (most recent call last):
  File "/usr/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/path/Sublist3r/sublist3r.py", line 565, in run
    domain_list = self.enumerate()
  File "/path/Sublist3r/sublist3r.py", line 625, in enumerate
    token = self.get_csrftoken(resp)
  File "/path/Sublist3r/sublist3r.py", line 620, in get_csrftoken
    token = csrf_regex.findall(resp)[0]
TypeError: expected string or buffer


```
